doc/rtd/development/dir_layout.rst

```diff
@@ -3,7 +3,7 @@
 Directory layout
 ****************
 
-``Cloud-init``'s directory structure is somewhat different from a regular
+Cloud-init's directory structure is somewhat different from a regular
```

Comment:
We should remove the directory listing listed immediately after this line. It is woefully out of date.